src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -187,6 +187,52 @@ def resource_exists(cli_ctx, subscription, resource_group, name, namespace, type
     return existing
 
 
+def register_global_policy_argument(cli_ctx):
+
+    def add_global_policy_argument(_, **kwargs):
+        command_table = kwargs.get('commands_loader').command_table
+
+        if not command_table:
+            return
+
+        class ChangeReferenceAction(argparse.Action):  # pylint:disable=too-few-public-methods
+
+            def __call__(self, parser, namespace, value, option_string=None):
+                # save change reference to CLI context
+                cmd = getattr(namespace, 'cmd', None) or getattr(namespace, '_cmd', None)
+                cmd.cli_ctx.data['_change_reference'] = value
+
+        class AcquirePolicyTokenAction(argparse.Action):  # pylint:disable=too-few-public-methods
+
+            def __call__(self, parser, namespace, value, option_string=None):
+                # save change reference to CLI context
+                cmd = getattr(namespace, 'cmd', None) or getattr(namespace, '_cmd', None)
+                cmd.cli_ctx.data['_acquire_policy_token'] = True
+
+        for command in command_table.values():
+            if command.name.split()[-1] in ['list', 'show']:
+                continue
+
+            change_reference_kwargs = {
+                'help': 'The related change reference ID for this resource operation',
+                'arg_group': 'Global Policy',
+                'action': ChangeReferenceAction,
+            }
+            acquire_policy_token_kwargs = {
+                'help': 'Acquiring an Azure Policy token automatically for this resource operation',
+                'arg_group': 'Global Policy',
+                'nargs': 0,
```

Comment:
According to https://docs.python.org/3/library/argparse.html#nargs, `nargs=0` can indeed be used, but `store_true` is preferred: https://docs.python.org/3/library/argparse.html#action

Reply:
Because  I need to store the value to `cli_ctx.data['_acquire_policy-token']` instead of to `namespace.acquire_policy_token`  so I've defined `action: AcquirePolicyTokenAction` already. That's why I'm not using `action: store_true`.

I defined `nargs=0` and set the value always to `True` in `AcquirePolicyTokenAction` to achieve the same as `store_true` action 😉